src/index.html

```diff
@@ -9,6 +9,9 @@
   <!-- prevent render loop from initializing on load -->
   <script>
     window['SohoConfig'] = { renderLoop: { noAutoStart: true }};
+
```

Comment:
Still not need with the latest PR where i removed this code?

Reply:
@tmcconechy but that stuff is for the renderLoop, did you that get fixed somehow where we don't need it?